images/capi/ansible/windows/roles/systemprep/tasks/main.yml

```
@@ -111,7 +111,7 @@
     type: string
 
 - name: Create SSH program data folder
-  win_shell: mkdir "$env:ProgramData\ssh"
+  win_shell: If (-Not (Test-Path -Path "$env:ProgramData\ssh")) { mkdir "$env:ProgramData\ssh" }
   ignore_errors: yes
```

id remove the ignore errors now because if it cant create then it should fail really